src/containers/markericon.jsx

```diff
@@ -7,61 +7,49 @@ import ICONS from './icons';
 // Using L because a component returns React.createElement() instead of L.geoJSON(..., { pointToLayer }
 // Reference: https://github.com/PaulLeCam/react-leaflet/issues/234
 
-const setMarker = (properties, latlng) => {
-  const orderType = {
-    Basilian: L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(<Icon icon={ICONS.MONASTERY} />),
-      }),
-    }),
-    'Augustinian Canons': L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(<Icon icon={ICONS.MONASTERY} />),
-      }),
-    }),
-    'Knights of the Hospital of Saint John of Jerusalem': L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(<Icon icon={ICONS.MONASTERY} />),
-      }),
-    }),
-    Benedictine: L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(<Icon icon={ICONS.MONASTERY} />),
-      }),
-    }),
-    Cistercian: L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(<Icon icon={ICONS.MONASTERY} />),
-      }),
-    }),
-    Cluniac: L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(<Icon icon={ICONS.MONASTERY} />),
-      }),
-    }),
-    'Knights Templar': L.marker(latlng, {
-      icon: L.divIcon({
-        html: ReactDOMServer.renderToString(
-          <Icon icon={ICONS.MONASTERY} translate="-6,-41" scale="1,-1" />
-        ),
-      }),
+const orderLookup = order => {
+  switch (order) {
+    case 'Basilian':
+      return { color: 'blue', scale: '1,-1', translate: '-6,-41' };
```

Comment:
Only return the props that change, depending on the order. So the returned object should only contain color.

Reply:
Moved scale and translate to a different variable. Need them for all monastery so I made a placeTypeLookup. Also going to pass the Icon through there too for future place types.